store/splus.js

```diff
@@ -165,12 +165,17 @@ export const getters = {
 };
 
 export const mutations = {
-  addLectures(state, { lectures, week }) {
+  /**
+   * Add given lectures to the state,
+   * paying respect to currently active whitelist.
+   */
+  setLectures(state, { lectures, week }) {
+    const whitelist = state.schedule.whitelist;
+    const filteredLectures = !!whitelist ? lectures.filter(
```

Comment:
wieso machst eig !!?

Reply:
!!! könnte existieren + nicht falsch sein ! ! !

Kurz für `whitelist !== undefined`

https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript
bang bang you're boolean

Reply:
short schneefux

Reply:
Kann man mögen, muss man nicht
Entscheiden wir durch Mehrheit, was sagst du @BrainConverter?

Reply:
ich habe kein Problem damit, hatte nur kein Plan was das ist ^^ -- ist es denn effizienter als die anderen?

Reply:
nö
sieht aber cool aus

Reply:
H4CK3RB0Y3